### ISSUE TITLE
GitHub action to automatically assign a label needs-triage

### ISSUE THREAD

**Issue by lusloher** [jina-ai/dashboard#228]
Task: GitHub action to automatically assign a label `needs-triage` to all new GitHub issues from community members. This is basically any issue created by another not in the Jina organization.

<p></p>

This should be set up for the following repos:  core repo, docs repo, hub, dashboard, jinabox, examples, cloud ops.

<p></p>

**Only applied to the issues from outside the organization.**

**Comment by lusloher** [comment]
Link to feature: https://jinaai.productboard.com/feature-board/planning/features/7160546

**Comment by FionnD** [comment]
Did some research, I think we can just use this? https://github.com/tunnckoCore/triage-new-issues 

What do you think @Roshanjossey ? 

It's actually ok that it's added to every issue, Jina people can just remove it manually.

**Comment by Roshanjossey** [comment]
@FionnD, I tried https://github.com/tunnckoCore/triage-new-issues, for a different project. It didn't work for me.

**Comment by Roshanjossey** [comment]
I'm thinking about using the solution here https://community.openhab.org/t/solved-how-to-assign-an-issue-on-github-to-me-how-to-merge-rebase-to-upstream/115559. Is that okay?

**Comment by FionnD** [comment]
I found a solution to this. 

**Solution:** 
[This action ](https://github.com/marketplace/actions/regex-issue-labeler) can be altered to match all new issues by inserting a REGEX that catches all text. It will then label any new issue created with needs_triage. 

**Steps:** 
These steps need to be taken for each of the following repos: Jina, Docs, hub, box, dashboard, examples, internal_tasks
1. Check the label needs-triage exists in the repo. 
2. Open a PR that contains the following: 
3. A file in the directory `.github/workflows/labeler.yml` which contains the following code: 
```name: "Issue Labeler"
on:
  issues:
    types: [opened, edited]

jobs:
  triage:
    runs-on: ubuntu-latest
    steps:
    - uses: github/issue-labeler@v2.3
      with:
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
        configuration-path: .github/labeler.yml
        enable-versioned-regex: 0 
```
4. A file in the directory `.github/labeler.yml` which contains the following code: 
```
# Add/remove 'needs-triage' label for all new issues that contain text in their body. 
needs-triage:
    - '(.)'
```
5. Once the PR is approved, add an issue yourself and check it has been correctly labeled. If so, delete the issue. 

It would make sense to do this for the docs repo first to double-check my approach. Open a PR, once it's approved, add a issue yourself and check it has been correctly assigned the label.
